.github/workflows/gradle.yml

```
@@ -9,7 +9,7 @@ jobs:
 
     strategy:
       matrix:
-        java: [ '17', '21', '24', '25-ea' ]
+        java: [ '17', '21', '25' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)